app/src/pages/income.tsx

```diff
@@ -1,12 +1,9 @@
 import incomeData from '@public/data/income.json'
```

Comment:
Notes for reviewers: The Income page (`/income`) is an editable form page in the form wizard flow. The back link routes to `/eligibility` and the call to action button routes the user to `/choose-clinic`. In order for a user to be allowed onto this page, they must have filled out the `/eligibility` page with valid data. 

The user is only routed to this page if they indicate that they are not enrolled in any qualifying adjunctive programs (such as medicaid). See https://github.com/navapbc/wic-mt-demo-project-eligibility-screener/pull/142/files#r1007690941



![Screen Shot 2022-10-28 at 2 15 45 AM](https://user-images.githubusercontent.com/67701/198551672-d00bb9d6-b1e0-4919-821e-2c6914446e16.png)